src/aave-v2/SupplyVaultBase.sol

```diff
@@ -196,5 +217,5 @@ abstract contract SupplyVaultBase is ISupplyVaultBase, ERC4626UpgradeableSafe, O
     /// @dev This empty reserved space is put in place to allow future versions to add new
     /// variables without shifting down storage in the inheritance chain.
     /// See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
-    uint256[47] private __gap;
+    uint256[48] private __gap;
```

Comment:
The gap should be decreasing, not increasing I think?

Reply:
This is to take into account the wrong number before

Reply:
I don't think that it is breaking if upgraded with this. We need to make sure that it is not, otherwise I'm for this change.